src/lib.rs

```diff
@@ -1077,20 +1078,31 @@ impl<'a> UncompressedHeader for BitWriter<'a, BE> {
         self.write_frame_size(fi)?;
         self.write(1,0)?; // don't use frame ids
         self.write(1,0)?; // use_128x128_superblock = 0
+        self.write_bit(true)?; // disable intra edge filter
```

Comment:
For a record,
I see that TD-Linux has revised existing uncompressed_header() and sequence_header(), not using obu_frame_header() and its helper() or sequence_header2(). 
And TD-Linux stated in #daala IRC before my comment here that : "btw my plan is once we hit v1.0.0, I will swap out the old sequence and uncompressed header functions with your 2 versions"